tensorrt_llm/_torch/pyexecutor/mamba_cache_manager.py

```
@@ -184,22 +271,150 @@ def get_state_indices(self) -> torch.Tensor:
+
+        accepted_conv_state = intermediate_conv_window_cache[:,
+                                                             src_state_indices,
+                                                             last_steps]
+        conv_states[:, dst_state_indices, :] = accepted_conv_state.to(
+            conv_states.dtype, copy=False)
+
+    def update_resources(
+            self,
+            scheduled_batch: ScheduledRequests,
+            attn_metadata: "AttentionMetadata" = None,
+            kv_cache_dtype_byte_size: float = None,
```

Ruff (0.14.11)

Unused method argument: `kv_cache_dtype_byte_size`

(ARG002)